src/main/play-doc/operation/ConfigurationRef.md

```diff
@@ -249,6 +309,53 @@ conductr {
     membership-settling-time = 5 seconds
   }
 
+  # Mesos related settings.
+  # There are only used if the 'conductr.mode' is set to 'mesos'
+  mesos-scheduler-client {
+    # The address of the Mesos master instance.
+    # The address can be either a Zookeeper or Mesos master address.
+    # A Zookeeper address is typically used when running multiple Mesos master instances (High-availabilty mode)
+    # An address directly to the Mesos master is used in standalone mode.
+    # Zookeeper formats:
+    #   zk://host1:port1,host2:port2,.../path
+    #   zk://username:password@host1:port1,host2:port2,.../path
+    #   file:///path/to/file (where file contains one of the above)
+    # Mesos standalone format:
+    #   host:port
+    master = "127.0.0.1:5050"
+
+    # ConductR framework info
+    framework {
+      name = "conductr"
+      checkpoint = false
+      user = "conductr"
+    }
+
+    mesos-executor {
+      # The start command of a Mesos executor (the conduct agent) that gets invoked on executor startup.
+      start-command = "$(pwd)/conductr-agent-*/bin/conductr-agent"
+
+      # Log level of the ConductR agent. Defaults to the loglevel of the ConductR core node.
+      loglevel = ${akka.loglevel}
+
+      # When supplied this represents the URIs required by a ConductR agent, including the agent itself.
+      # When using Mesos, this is used to supply the Mesos Fetcher with the agent's packages so that it may
+      # be dynamically downloaded.
+      agent-uris = []
+
+      # Controls whether downloads of the agent are cached. The default is to cache downloads for
+      # improved performance.
+      agent-download-cache = on
+
+      # Resources needed to start a Mesos executor that runs a ConductR agent
+      resources {
+        cpu        = 1.0
```

Comment:
The resources have changes with the PR https://github.com/typesafehub/conductr/pull/1269.

New values:

```
      resources {
        cpu        = 0.1
        memory     = 268400000
        disk-space = 1000000000
      }
```